src/library/scala/collection/mutable/HashMap.scala

```diff
@@ -72,6 +72,17 @@ extends AbstractMap[A, B]
     else Some(e.value)
   }
 
+  override def getOrElseUpdate(key: A, defaultValue: => B): B = {
```

Comment:
I'm not sure this is a binary-compatible change.  @retronym - is this?  I would not intuitively have thought it would be with the new trait encoding, but I'm not sure of the new rules.